AdminClient/src/app/customers/customers.component.html

```diff
@@ -0,0 +1,82 @@
+<div class="container"
+     fxLayout
+     fxLayout.xs="column"
+     fxLayoutAlign="center stretch"
+     fxLayoutGap="10px"
+     fxLayoutGap.xs="0">
+
+  <div id="table">
+    <table class="table table-striped">
+      <thead>
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Category</th>
+        <th>Telephone</th>
+        <th>Email</th>
+        <th>Website</th>
+        <th>Contact</th>
+        <th>Address</th>
+        <th>Town/City</th>
+        <th>County</th>
+        <th>Country</th>
+        <th>Postcode</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr *ngFor="let customer of customers">
+        <td>{{ customer.id }}</td>
+        <td>{{ customer.name }}</td>
+        <td>{{ customer.category }}</td>
+        <td>{{ customer.telephone }}</td>
+        <td>{{ customer.email }}</td>
+        <td>{{ customer.website }}</td>
+        <td>{{ customer.contact }}</td>
+        <td>{{ customer.address }}</td>
+        <td>{{ customer.townOrCity }}</td>
+        <td>{{ customer.county }}</td>
+        <td>{{ customer.country }}</td>
+        <td>{{ customer.postcode }}</td>
+      </tr>
+      </tbody>
+    </table>
+  </div>
+
+</div>
+
+<div class="container"
```

Comment:
If possible transfer this to a toolbar format rather than just throwing them out there. Again see what the Angular team offers you out of the box https://material.angular.io/components/toolbar/overview so it wont take you time to set it up. Also see how you will integrate this items together i.e Will this be a pop-up? Will this cover the entire screen?